pegasus/sites.v3/code.org/public/about/leadership/index.md.erb

```diff
@@ -15,10 +15,13 @@ style_min: true
 
 <%= view :about_headshots, people:DB[:cdo_leaders].where(kind_s:'LT') %>
 
-## Global Council & Education Leadership Council
+## Education Leadership Council
 
-We also work with two leadership councils to inform our work. [Learn more here](/about/education-leadership-council).
+<%= view :about_headshots, people:DB[:cdo_leaders].where(kind_s:'council') %>
 
 ## We're grateful for the advice and help of so many leaders over the years:
 
 <%= view :about_people, people:DB[:cdo_leaders].where(kind_s:'grateful') %>
+
+## We're grateful for the advice and help of so many leaders over the years:
```

Comment:
This heading can be removed, looks like a duplicate from the section above:
![Screenshot 2023-07-24 at 2 09 20 PM](https://github.com/code-dot-org/code-dot-org/assets/9256643/f3502f51-458b-4174-b910-211f2e97fa49)